truncate/table_schema.go

```diff
@@ -177,7 +188,7 @@ func targetFilterTableSchemas(tables []*tableSchema, targetTables []string) []*t
 
 // excludeFilterTableSchemas filters tables with given excludeTables.
 // If excludeTables is empty, it returns all tables.
-// When an exclude table is cascade deletable, its parent table is also excluded.
+// When an exclude table is cascade deletable, its ancestors tables are also excluded.
```

Comment:
The existing code was ambiguous in its handling of parent-child and ancestor-descendant relationships.
I refactored the code to clear the differences between them.